src/app/service/question.service.ts

```diff
@@ -21,7 +21,7 @@ export class QuestionService {
     if (document.cookie) {
```

Comment:
No need to add condition here as document.cookie return string and condition will always right